### ISSUE TITLE
When I try to rename some equipment to something else. It automatically changes back to what it was before.? 

### ISSUE THREAD

**Issue by gminton86** [reef-pi/reef-pi#824]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Device [e.g. Pi3]
 - reef-pi version [e.g. 2.0.1]

**Additional context**

**Comment by gminton86** [comment]
ok here is the specifics. Say I make an equipment and name it heater and then control it in the temperature section as the heat controller and enable it with a set point at 80F. If I go in and turn the temp control to disabled and then try to reuse the equipment that I originally had named heater. It will keep renaming it heater even though it is disabled.

**Comment by gminton86** [comment]
to get past it I had to completely remake my setup and start from scratch I could not figure out how to get it to go away and it kept being controlled by the temp so it was turning off and on my newly setup equipment that was using the same connector it was originally on. and turning my refugium light off and on from the temp of the water.

**Comment by zekth** [comment]
No error in logs?

**Comment by ranjib** [comment]
@gminton86 do you have any timer or macro using the equipment? If you do. Disable those. Then update the equipment and then re-enable the timer. I think its a bug, but lets verify

**Comment by gminton86** [comment]
no errors and there was no timers at all as my light has a built in timer and no macros that controlled it I looked for that first before I reset my system. The steps I shared are exactly how it happened to me and I haven't tried to do it again. And I don't let the reef pi control my heater anymore on this new setup. But I will get another pi I have laying around and set it up and see if it does it again to figure out the exact reason. To pinpoint a solution. Possibly?
